bin/git_log_to_prs.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'optimist'
+
+opts = Optimist.options do
+  opt :file, "File containing output from running show_commit_history", :type => :string,  :required => true
```

Comment:
Is this necessary, as I'd expect things can just write to STDOUT and then be redirected into a file?

Reply:
Oh I see, this is taking the output of show_commit_history?  Instead, can we just run show_commit_history and capture the output and/or create a helper class and just call it?  Seems strange to make the caller run 2 different tools.

Reply:
In fact, if you just do it inline (reusing what's in show_commit_history, then there's no parsing of the output command since each iteration of the loop will just have the repo available as an object  (you'd still need to parse the git output)

Reply:
How about this becomes a new option for show_commit_history, so user can choose the output format?

Reply:
I put together #159